definitions/private-law/json/CaseEventToComplexTypes/ManageOrders/FL404/CaseEventToComplexTypes.json

```diff
@@ -0,0 +1,1099 @@
+  },
+  {
+    "LiveFrom": "20/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "FL400",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "fl404CustomFields",
+    "ListElementCode": "fl404bIsPowerOfArrest6",
+    "EventElementLabel": " ",
+    "FieldDisplayOrder": "83",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "manageOrdersOptions=\"createAnOrder\" AND  fl404CustomFields.fl404bRespondentOtherInstructions=\"other2\""
+  },
+  {
+    "LiveFrom": "20/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "FL400",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "fl404CustomFields",
+    "ListElementCode": "fl404bDateOrderMade",
+    "EventElementLabel": "Date order made",
+    "FieldDisplayOrder": "84",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "createSelectOrderOptions=\"powerOfArrest\" OR createSelectOrderOptions=\"nonMolestation\" OR createSelectOrderOptions=\"occupation\""
+  },
+  {
+    "LiveFrom": "20/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "FL400",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "fl404CustomFields",
+    "ListElementCode": "fl404bDateOrderEnd",
+    "EventElementLabel": "Date order ends",
+    "FieldDisplayOrder": "85",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "createSelectOrderOptions=\"powerOfArrest\" OR createSelectOrderOptions=\"nonMolestation\" OR createSelectOrderOptions=\"occupation\""
+  },
+  {
+    "LiveFrom": "20/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "FL400",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "fl404CustomFields",
+    "ListElementCode": "fl404bDateOrderEndTime",
+    "EventElementLabel": "Time",
+    "FieldDisplayOrder": "86",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "createSelectOrderOptions=\"powerOfArrest\" OR createSelectOrderOptions=\"nonMolestation\" OR createSelectOrderOptions=\"occupation\""
+  },
+  {
+    "LiveFrom": "20/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "FL400",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "fl404CustomFields",
+    "ListElementCode": "fl404bDateOfNextHearing",
+    "EventElementLabel": " ",
+    "FieldDisplayOrder": "87",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "createSelectOrderOptions=\"occupation\""
+  },
+  {
+    "LiveFrom": "20/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "FL400",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "fl404CustomFields",
+    "ListElementCode": "fl404bTimeOfNextHearing",
+    "EventElementLabel": "Time",
+    "FieldDisplayOrder": "88",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "createSelectOrderOptions=\"occupation\""
+  },
+  {
+    "LiveFrom": "20/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "FL400",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "fl404CustomFields",
+    "ListElementCode": "fl404bTimeEstimate",
```

Comment:
'fl404bTimeEstimate' name has fl404b, but this field is commonly being used for FL404, FL404A & FL406. This creates little confusion. Can we use some other generic element name like FL400TimeEstimate